### PR TITLE
add facts views to host and inv host detail views

### PR DESCRIPTION
**Pull request by jlmitch5**
link #5907 

There were also some string changes to the option for enabling fact storage that is hopefully a bit clearer.  Thanks to @trahman73 @tvo318 @chrismeyersfsu and @wenottingham for the help there:

Screen Shot 2020-03-13 at 2 01 05 PM
Screen Shot 2020-03-13 at 2 00 46 PM

### CODE REVIEW

File: awx/ui_next/package-lock.json [hunk 1]
```diff
@@ -220,7 +220,7 @@
         },
```
Comment (AlexSCorey) [inline awx/ui_next/package-lock.json]: We probably want to revert these changes so that we are using the secure url instead of the insecure 1.

File: awx/ui_next/src/components/CodeMirrorInput/CodeMirrorInput.jsx [hunk 2]
```diff
@@ -63,6 +64,7 @@ function CodeMirrorInput({
   readOnly,
   hasErrors,
   rows,
+  fullHeight,
```
Comment (AlexSCorey) [inline awx/ui_next/src/components/CodeMirrorInput/CodeMirrorInput.jsx:67]: I assume this is not a required prop, right?  We should probably add this to prop types/default props below.

File: awx/ui_next/src/screens/Inventory/InventoryHostFacts/InventoryHostFacts.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+function InventoryHostFacts({ i18n, host }) {
+  const { result: facts, isLoading, error, request: fetchFacts } = useRequest(
+    useCallback(async () => {
+      const [{ data: factsObj }] = await Promise.all([
+        HostsAPI.readFacts(host.id),
+      ]);
+      return JSON.stringify(factsObj, null, 4);
+    }, [host]),
+    '{}'
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/Inventory/InventoryHostFacts/InventoryHostFacts.jsx:21]: Why is this in quotes?

File: awx/ui_next/src/screens/Inventory/shared/data.hostFacts.json [hunk 4]
```diff
@@ -0,0 +1,1249 @@
+{
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/Inventory/shared/data.hostFacts.json:1]: Seems like you have a file of essentially the same name in `/hosts/` could we remove 1 of them?

File: awx/ui_next/package-lock.json (hunk 1 above)
Comment (jlmitch5) [reply awx/ui_next/package-lock.json]: didn't realize I commited package-lock.json at all!  That was unintentional, I'll remove that.

File: awx/ui_next/src/components/CodeMirrorInput/CodeMirrorInput.jsx (hunk 2 above)
Comment (jlmitch5) [reply awx/ui_next/src/components/CodeMirrorInput/CodeMirrorInput.jsx:67]: It is good call.

File: awx/ui_next/src/screens/Inventory/InventoryHostFacts/InventoryHostFacts.jsx (hunk 3 above)
Comment (jlmitch5) [reply awx/ui_next/src/screens/Inventory/InventoryHostFacts/InventoryHostFacts.jsx:21]: because code mirror expects the value to be a string.

File: awx/ui_next/src/screens/Inventory/shared/data.hostFacts.json (hunk 4 above)
Comment (jlmitch5) [reply awx/ui_next/src/screens/Inventory/shared/data.hostFacts.json:1]: agreed this is weird and feels like unnecessary duplication, but we don't want to share across screens.

**Review by jakemcdermott — APPROVED** [review]
